portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -126,4 +137,23 @@ private String getBody(HttpServletRequest request) throws IOException {
 
         return "";
     }
+
+    private double analyzeSentiment(String comment) throws IOException {
+        try (LanguageServiceClient languageService = LanguageServiceClient.create()) {
+            Document document = Document
+                .newBuilder()
+                .setContent(comment)
+                .setType(Document.Type.PLAIN_TEXT)
+                .build();
+
+            double sentiment = (double) languageService
+                .analyzeSentiment(document)
+                .getDocumentSentiment()
+                .getScore();
+
+            languageService.close();
```

Comment:
Because you added the try-with-resources on line 142, it is no longer necessary to call `languageService.close()`. In Java, classes that implement [AutoCloseable](https://docs.oracle.com/javase/7/docs/api/java/lang/AutoCloseable.html?is-external=true) (which LanguageServiceClient [does](https://googleapis.dev/java/google-cloud-language/latest/com/google/cloud/language/v1/LanguageServiceClient.html)) will have their `close()` methods called automatically upon exiting the `try` block.